samples/inspect.js

```
@@ -221,13 +221,150 @@ async function inspectGCSFile(
+          },
+        },
+      ],
     },
   };
 
+  try {
+    // Create a GCS File inspection job and wait for it to complete
+    const [topicResponse] = await pubsub.topic(topicId).get();
+    // Verify the Pub/Sub topic and listen for job notifications via an
+    // existing subscription.
+    const subscription = await topicResponse.subscription(subscriptionId);
+    const [jobsResponse] = await dlp.createDlpJob(request);
+    // Get the job's ID
+    const jobName = jobsResponse.name;
+    // Watch the Pub/Sub topic until the DLP job finishes
+    await new Promise((resolve, reject) => {
+      const messageHandler = message => {
+        if (message.attributes && message.attributes.DlpJobName === jobName) {
+          message.ack();
+          subscription.removeListener('message', messageHandler);
+          subscription.removeListener('error', errorHandler);
+          resolve(jobName);
+        } else {
+          message.nack();
+        }
+      };
+
+      const errorHandler = err => {
+        subscription.removeListener('message', messageHandler);
+        subscription.removeListener('error', errorHandler);
+        reject(err);
+      };
+
+      subscription.on('message', messageHandler);
+      subscription.on('error', errorHandler);
+    });
+
+    setTimeout(() => {
+      console.log(`Waiting for DLP job to fully complete`);
+    }, 500);
+    const [job] = await dlp.getDlpJob({name: jobName});
+    console.log(`Job ${job.name} status: ${job.state}`);
+
+    const infoTypeStats = job.inspectDetails.result.infoTypeStats;
+    if (infoTypeStats.length > 0) {
+      infoTypeStats.forEach(infoTypeStat => {
+        console.log(
+          `  Found ${infoTypeStat.count} instance(s) of infoType ${
+            infoTypeStat.infoType.name
+          }.`
+        );
+      });
+    } else {
+      console.log(`No findings.`);
+    }
+  } catch (err) {
+    console.log(`Error in inspectGCSFile: ${err.message || err}`);
```

Let's drop the `try/catch`, and just let it throw if there's an error.

---

Most of the code samples in this repo use this `try/catch` pattern. Should I change other samples as well, or just this one, or none?

---

I need to do an all-up pass on this repo anyway, so feel free to ignore ;)